app/src/main/java/com/sdp13epfl2021/projmag/activities/ProjectInformationActivity.kt

```diff
@@ -182,10 +192,17 @@ class ProjectInformationActivity : AppCompatActivity() {
     private fun addVideoAfterDownloaded(videosLinks: List<String>) {
         videosLinks.forEach { link ->
             fileDB.getFile(link, projectDir, { file ->
-                addVideo(Uri.fromFile(file))
-            }, {
-                showToast("An error occurred while downloading a video.")
-            })
```

Comment:
If the videos couldn't be downloaded, won't it be better to have a message to inform the user ?

Reply:
I deleted it by accident
thank you